src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const lines = sourceString.split(';');
+
+  const result = lines.reduce((acc, curr) => {
+    const clean = curr.trim();
+
+    if (clean.includes(':')) {
+      const parts = clean.split(':', 2);
```

Comment:
```suggestion
      const [key, value] = clean.split(':', 2);
```
You can do it like that -immediately destructure it